pkg/proxy/proxy.go

```diff
@@ -212,16 +216,16 @@ func (p *Proxy) RoundTrip(req *http.Request) (*http.Response, error) {
 	// address.
 	if p.options.ExtraUserHeadersClientIPEnabled {
 		klog.V(6).Infof("adding impersonate extra user header %s: %s (%s)",
-			UserHeaderClientIPKey, req.RemoteAddr, reqCpy.RemoteAddr)
+			UserHeaderClientIPKey, clientAddress, clientAddress)
```

Comment:
Previous version used `reqCpy` for the second argument, this version duplicates it?

Reply:
reqCpy is going away

Reply:
:+1: but do you need to print the same text twice..? `adding impersonate extra user header %s: %s (%s)` why put the same text in brackets immediately after including it the first time?

Reply:
All log output related to a request is paired with the IP address. It _just so happens_ that the log output itself is also related to that IP address. I think in a log stream it doesn't look that odd